src/System.Runtime.Serialization.Formatters/src/System/Runtime/Serialization/Formatters/Binary/Converter.cs

```diff
@@ -30,7 +30,7 @@ internal static class Converter
         internal static readonly Type s_typeofUInt64 = typeof(ulong);
         internal static readonly Type s_typeofObject = typeof(object);
         internal static readonly Type s_typeofSystemVoid = typeof(void);
-        internal static readonly Assembly s_urtAssembly = s_typeofString.Assembly;
+        internal static readonly Assembly s_urtAssembly = Assembly.Load("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089");
```

Comment:
Yes it is the shim from shims/manual/mscorlib.csproj. Aren't we already building it because of this: https://github.com/dotnet/corefx/blob/master/src/dirs.proj#L14 ?